### ISSUE TITLE
IMPORTANT: Show full title for deleted videos

### ISSUE THREAD

**Issue by davidhedlund** [youtube/api-samples#164]
# Example 1
For https://www.youtube.com/watch?v=A0WPjZwKhA

![screenshot from 2018-02-23 01 19 40](https://user-images.githubusercontent.com/6416073/36571574-a69cd508-1837-11e8-98c4-a5b20ce99d1f.png)

The original title was:
> Artifact303 ~ Life Support System ᴴᴰ

But I didn't get any text match when I searched "Artifact303" in the HTML source.

# Example 2

For https://www.youtube.com/watch?v=W1HiCm__NL0

![screenshot from 2018-02-23 01 19 18](https://user-images.githubusercontent.com/6416073/36571584-b3d350ee-1837-11e8-9f2b-1613aa286444.png)

Only the first 20 character are displayed in this example. I didn't find anything more than "EPHEDRA - Moro Ro..." in the HTML source.

That means that it's impossible to find out what the original track was for artists with longer names if it's not stored in the Wayback Machine. This can result in hundreds of lost favorite titles for users that has made personal best of playlists.

Please show the full title.

**Comment by davidhedlund** [comment]
@jeffposnick Hi, I hope you can help me. I take this very serious. Have I reported this to the right place?

**Comment by megapro17** [comment]
any progress?

**Comment by davidhedlund** [comment]
Lost another video today since I only got the URL but not the title for https://www.youtube.com/watch?v=3oUMkoKEcyc

**Comment by ninsky** [comment]
www.RecoverMy.video is your friend :)

**Comment by schoerg** [comment]
@ninsky found this while seraching. unfortunateley recovermyvideo does NOT recover the video. It just says "yeah we recover your videos from now on".

**Comment by ninsky** [comment]
@schoerg, correct. RecoverMy.Video recoveres the title of the video as requested on this thread ("Show full title for deleted videos").

**Comment by schoerg** [comment]
Excepct it did not do that. I would have been fine with the title only. It was literally "we couldn't recover the titles, but from now on we can".
